### PR TITLE
GSS bz-2274193 noobaa db cleartext postgres password

### PR DESCRIPTION
**Pull request by OdedViner**
https://bugzilla.redhat.com/show_bug.cgi?id=2274193

### CODE REVIEW

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class TestNoobaaSecurity(BaseTest):
+    """
+    Test Noobaa Security
+    """
+
+    @green_squad
+    @bugzilla("2274193")
+    def test_noobaa_db_cleartext_postgres_password(self):
+        """
+        1.Get noobaa deb pod
```
Comment (sagihirshfeld) [inline tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:23]: Small typo:

```suggestion
        1.Get noobaa db pod
```

Comment (OdedViner) [reply tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py]: done

**Comment by OdedViner** [comment]
@hnallurv @PrasadDesala the test failed on ODF4.15.2 [as expected]

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.framework.testlib import tier2, BaseTest, bugzilla
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+
+
```
Comment (mashetty330) [inline tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:8]: Move this test under MCG.

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2274193")
```
Comment (mashetty330) [inline tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:13]: we dont need this redundant bugzilla markers

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+class TestNoobaaSecurity(BaseTest):
+    """
+    Test Noobaa Security
+    """
+
+    @green_squad
```
Comment (mashetty330) [inline tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:19]: please move this test under mcg and add red_squad, mcg markers

**Review by parikshithb — COMMENTED** [review]
lgtm, need to rectify other comments.

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py (hunk 4 above)
Comment (OdedViner) [reply tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:19]: done

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py (hunk 3 above)
Comment (OdedViner) [reply tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:13]: why?

File: tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py (hunk 2 above)
Comment (OdedViner) [reply tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py:8]: done

**Comment by sagihirshfeld** [comment]
@OdedViner now that this is a Red Squad test you need to add the `mcg` decorator (in this case) for the PR checks to pass:
```
  INTERNALERROR> ocs_ci.ocs.exceptions.MissingDecoratorError: 
  INTERNALERROR> The following tests are missing either the @mcg or @rgw decorators: {
  INTERNALERROR>     "test_noobaa_db_cleartext_postgres_password": "/home/runner/work/ocs-ci/ocs-ci/tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py"
  INTERNALERROR> }
  INTERNALERROR> 
  INTERNALERROR> Red squad tests are required to be decorated with either mcg or rgw. Please add either depending on the tests's focus.
```

**Comment by OdedViner** [comment]
> @OdedViner now that this is a Red Squad test you need to add the `mcg` decorator (in this case) for the PR checks to pass:
> 
> ```
>   INTERNALERROR> ocs_ci.ocs.exceptions.MissingDecoratorError: 
>   INTERNALERROR> The following tests are missing either the @mcg or @rgw decorators: {
>   INTERNALERROR>     "test_noobaa_db_cleartext_postgres_password": "/home/runner/work/ocs-ci/ocs-ci/tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py"
>   INTERNALERROR> }
>   INTERNALERROR> 
>   INTERNALERROR> Red squad tests are required to be decorated with either mcg or rgw. Please add either depending on the tests's focus.
> ```

done

**Review by sagihirshfeld — DISMISSED** [review]

File: tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+from ocs_ci.framework.testlib import tier2, BaseTest, bugzilla
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+
+
+log = logging.getLogger(__name__)
+
+
```
Comment (PrasadDesala) [inline tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py:11]: polarion id missing

File: tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+@tier2
+@bugzilla("2274193")
+class TestNoobaaSecurity(BaseTest):
+    """
+    Test Noobaa Security
+    """
+
+    @mcg
+    @red_squad
+    @bugzilla("2274193")
```
Comment (PrasadDesala) [inline tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py:21]: Remove the duplicate marker, you added the bugzilla already above

Comment (OdedViner) [reply tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py:21]: https://github.com/red-hat-storage/ocs-ci/pull/9806#issuecomment-2119168967

File: tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py (hunk 5 above)
Comment (OdedViner) [reply tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py:11]: done

**Review by sagihirshfeld — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**